video_generation/video.py

```diff
@@ -498,17 +498,17 @@ def create(video_directory, post, thumbnails):
 
                     txt_clip = (
                         TextClip(
-                            current_clip_text,
+                            accepted_comment.body,
                             font=settings.text_font,
                             fontsize=settings.text_fontsize,
                             color=settings.text_color,
                             size=txt_clip_size,
                             kerning=-1,
                             method="caption",
-                            # bg_color=settings.text_bg_color,
+                            bg_color=settings.text_bg_color,
```

Comment:
`commentstyle="text"` logic won't attempt to screenshot the reddit comment, instead it will just get the comment body from the reddit API and moviepy will create a text comment in the video, this should be improved to make it look more like a reddit screenshot comment.